docs/src/language_servers.md

```diff
@@ -627,6 +627,23 @@ Follow installation instructions on [LSP-metals](https://github.com/scalameta/me
     }
     ```
 
+### Solidity
+
+1. Install [github:NomicFoundation/hardhat-vscode](https://github.com/NomicFoundation/hardhat-vscode/tree/development/server) (follow instructions in the repo):
+
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"solidity"` client configuration to the `"clients"`:
```

Comment:
I'm guessing the user also has install some syntax for "solidity"?

Reply:
The `source.solidity` scope used for `"selector"` is not built-in from Sublime Text, so you would need to have the according syntax package from Package Control installed to make the language server start. The other config examples on this page link to the syntax packages.